server/db/dbConnect.js

```diff
@@ -2,6 +2,12 @@ import mongoose from "mongoose";
 
 const DB_URL = process.env.DB_URL;
 const DB_NAME = process.env.DB_NAME;
+const DB_CONNECTION_STRING = `${DB_URL}${DB_NAME}?retryWrites=true&w=majority`
+const DB_OPTS = {
+  bufferCommands: false,
+};
+
+export const DB_INFO = { DB_URL, DB_NAME, DB_CONNECTION_STRING, DB_OPTS };
```

Comment:
maybe export the `DB_INFO` variables separately and on an as needed basis. I don't think we need `DB_INFO` as another layer of abstraction for env vars
```js
export const DB_URL
export const DB_NAME
...
```

Reply:
Done 👍